src/Augurk.CommandLine/Commands/PublishCommand.cs

```diff
@@ -106,6 +113,38 @@ private void ExecuteUsingV1Api()
             }
         }
 
+        /// <summary>
+        /// Expands a list of feature file specifications by resolving wildcards.
+        /// </summary>
+        /// <param name="featureFiles">List of feature files specified by the user.</param>
+        /// <returns>Expanded set of file names.</returns>
+        private static IEnumerable<string> Expand(IEnumerable<string> featureFiles)
+        {
+            var expandedList = new List<string>();
+
+            foreach (var fileSpec in featureFiles)
+            {
+                if (fileSpec.Contains('?') || fileSpec.Contains('*'))
+                {
+                    // resolve wildcard
+                    var directory = Path.GetDirectoryName(fileSpec);
+                    var spec = Path.GetFileName(fileSpec);
+
+                    if (!string.IsNullOrEmpty(directory) && !string.IsNullOrEmpty(spec))
```

Comment:
I do not believe this check is sufficient. In your code the directory variable might be the one to end up with a __*__ or **?** character. In that case the Directory.GetFiles will throw an exception.

The same goes for the scenario where the Path does not exist.

I think the best option would be to call Directory.Exist. This ensures that no matter the user-input, no exceptions will be thrown.